angular-client/src/services/storage.service.ts

```diff
@@ -0,0 +1,17 @@
+import { DataValue, StorageMap } from 'src/utils/socket.utils';
+
+export default class Storage {
```

Comment:
services are like hooks yeah?

Reply:
yes but they can be more generic too, they can be hooks and other things that react has separate terms for